testutil/vault.go

```diff
@@ -198,7 +200,12 @@ func (tv *TestVault) Stop() {
 		tv.t.Errorf("err: %s", err)
 	}
 	if tv.waitCh != nil {
-		<-tv.waitCh
+		select {
+		case <-tv.waitCh:
+			return
```

Comment:
Do we want to assert that `waitCh` has no errors?

Reply:
we're not testing vault itself, so this is probably not super useful?

Reply:
If we aren't asserting on it, we can turn `waitCh` to be a channel of `interface{}`?  It's a bit confusing that we are sending errors in L174 to the channel for them to be ignored.